R/isobar-import.R

```diff
@@ -1422,8 +1422,14 @@ read.mzid <- function(filename) {
     ib.data <- filename
   else
     id.data <- read.delim(filename, sep="\t", stringsAsFactors=FALSE)
+
+  if (! "PepQValue" %in% colnames(id.data)) {
+    stop("Error: No q-values found in result files.\nPlease repeat the MSGF+ search using a\ntarget-decoy search (option -tda 1)")
```

Comment:
I think "Error: " should be omitted to match the other isobar error messages.
Also "result files" is not very specific; maybe something like "MSGF+ files"?